modules/mapbox/src/mapbox-overlay.js

```diff
@@ -0,0 +1,149 @@
+import {Deck} from '@deck.gl/core';
+import {getViewState} from './deck-utils';
+
+export default class MapboxOverlay {
```

Comment:
Nits:
- Add JSDoc before classes and method? - Quite helpful as these show up in vscode.
- Isn't react-map-gl already transitioned to typescript? Why add another untyped .js file?